### ISSUE TITLE
duplicate auth_basic when NGINX_AUTHENTICATION_TYPE=BASIC

### ISSUE THREAD

**Issue by terryzwt** [tiredofit/docker-nginx#13]
### Summary
Here is the docker-compose.yml I used.
```
version: '2.4'
services:

  nginx-app:
    image: tiredofit/nginx:latest
    container_name: nginx-app
    labels:
    labels:
      - traefik.enable=true
      - traefik.http.routers.nginx-app.rule=Host(`nginx.docker`)
      - traefik.http.services.nginx-app.loadbalancer.server.port=80
    volumes:
      - ./data:/www/html
      - ./logs:/www/logs
    environment:
      - VIRTUAL_HOST=nginx.docker
      - NGINX_MODE=normal
      - NGINX_AUTHENTICATION_TYPE=BASIC
        #- NGINX_AUTHENTICATION_TITLE=TERRY
      - NGINX_AUTHENTICATION_BASIC_USER1=user1
      - NGINX_AUTHENTICATION_BASIC_PASS1=password1
      - CONTAINER_NAME=nginx-app
      - CONTAINER_LOG_LEVEL=DEBUG
    networks:
      - proxy
    restart: always

networks:
  proxy:
    external: true
```

After docker compose up -d, see below errors, Looks the `auth_basic` has duplicated.
<img width="956" alt="image" src="https://user-images.githubusercontent.com/1829399/185345467-bbf4c58f-ef3c-4286-9aee-5f8768298dfa.png">

**Comment by tiredofit** [comment]
Thanks for this - This has been addressed in `6.1.11` 
I was performing inserting the authentication function twice when there was no custom configuration in place and it was relying on the auto generated `default.conf` file.

**Comment by terryzwt** [comment]
After try the new image, already fixed!
